Add contributor's guide (#23)
buildpacks/community: contributors/triage.md

# Triage

> Triaging is the act of managing issues, determining the appropriate response, and following up.

Reviewer: ```suggestion
Triaging is the act of managing issues, determining the appropriate response, and following up with the original poster to ensure the problem was solved.
```

### Actionable Issues

> An actionable issue is one that can be worked on such that the following is true:

Reviewer: ```suggestion
An actionable issue is one that can be worked on, such that the following is true:
```

> - Definition of done - that may be acceptance criteria or clear expectations.

Reviewer: ```suggestion
- A definition of done is provided - whether that is formal acceptance criteria or clear expectations.
```

> - Appropriate labels

Reviewer: ```suggestion
- Appropriate label(s) is present on the issue
```

    - `type/*`
    - `size/*` (if possible)
    - `status/ready`
    - `good first issue` (if applicable)

### Actions

> The following is a flow to assist contributors in assigning the appropriate labels in script format.

Reviewer: ```suggestion
The following is a flow to assist contributors in assigning the appropriate labels to an issue.
```

        - See below
- Is this something we want to do?
    - I don’t even understand this issue
        - Remove label `status/triage`
        - Add label `status/needs-information`
        - Comment in request for clarification
        - If activity is missing for 10 days, close
    - No - not valid / relevant for our project
        - Remove label `status/triage`
        - Add label `status/invalid`
        - Comment and close
    - Maybe - needs RFC
        - Remove label `status/triage`
        - Add label `status/requires-rfc`
        - Comment on suggestion for RFC
        - If no further activity for 10 days, close
    - Maybe - needs alignment
        - Remove label `status/triage`
        - Add label `status/discussion-needed`
        - Add appropriate `type/*`
    - Maybe - I can’t tell from the information in the issue
        - Remove label `status/triage`
        - Add label `status/needs-information`
        - Request further information from issue author
        - If no further activity for 10 days, close
    - Yes
        - Could this be worked on right now?
            - Yes
                - Remove label `status/triage`
                - Add label `status/ready`

>                 - Add appropriate `size/*` (if possible) and `type/*`

Reviewer: ```suggestion
                - Add appropriate `size/*` (if possible), `type/*`, and `good-first-issue` if it is relatively straightforward
```
